src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -147,8 +161,10 @@ void FlexDRWorker::initNetObjs_via(
 {
   auto net = via->getNet();
   nets.insert(net);
-  if (getRouteBox().intersects(via->getOrigin())) {
-    netRouteObjs[net].push_back(make_unique<drVia>(*via));
+  if (isRouteVia(via)) {
+    auto uVia = make_unique<drVia>(*via);
```

Comment:
Why is intersection testing no longer needed?

Reply:
@maliberty I test it in isRouteVia